execute/plugin.go

```diff
@@ -127,20 +128,12 @@ func getPendingExecutedReports(
 	ts time.Time,
 	lggr logger.Logger,
 ) (exectypes.CommitObservations, error) {
-	latestReportTS := time.Time{}
-	commitReports, err := ccipReader.CommitReportsGTETimestamp(ctx, dest, ts, 1000)
+	commitReports, err := ccipReader.CommitReportsGTETimestamp(ctx, dest, ts, 1000) // todo: configurable limit
```

Comment:
I thought we had a limit in the offchain config

Reply:
Seems not. In <= v1.5 we had a const in the code somewhere, I think that might also be fine vs. the magic 1K. Lets create a ticket in the performance backlog?